user_journey/views/funnel_analysis/client_properties.view.lkml

```diff
@@ -0,0 +1,58 @@
+view: client_properties {
+  derived_table: {
```

Comment:
I also tried simply using the `clients_last_seen` view from looker-hub. However that required to rename the existing `clients_last_seen` view in `cohort_analysis/` and I'm not sure how convenient it is to have all the `clients_last_seen` fields listed in the explore.

Reply:
We should be able to use [`from`](https://docs.looker.com/reference/explore-params/from-for-join) to specify the Looker view without creating a new one, and [`fields`](https://docs.looker.com/reference/explore-params/fields-for-join) to specify which fields to expose.

Alternatively, [extend](https://docs.looker.com/reference/view-params/extends-for-view) that view with this one, and just add those new measures you need at the bottom.